src/components/ProbeView.vue

```diff
@@ -0,0 +1,55 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+import { useProbeStore } from '@/src/store/probe';
+import { storeToRefs } from 'pinia';
+
+const probeStore = useProbeStore();
+const { probeData } = storeToRefs(probeStore);
+
+const formattedProbeItems = computed(() => {
+  if (!probeData.value) return [];
+  const sampleItems = probeData.value.samples.map((sample) => ({
+    label: sample.name,
+    value: sample.displayValue.join(', '),
```

Comment:
We should shorten the displayValue, since floats will show up as very long numbers.

Something like the following:
```
if Number.isInteger(value) return value
if 0 < Math.abs(value) < 1 return value.toExponential(2)
return value.toFixed(2)
```

Reply:
Yes.  I did this with `num.toPrecision` which is similar.  `toPrecision` won't work well for images with large `origin` values though.  

![image](https://github.com/user-attachments/assets/6a105cca-e346-4e06-836c-68cae2b00a79)

Reply:
I think toPrecision isn't the right thing to use here, because of the issue you mentioned and because we're not looking to show fixed-precision values. toFixed is a better approach, but truncates to 0 for sufficiently tiny values in [0,1].

Reply:
Ok, added a `shortenNumber` func with your code.

Reply:
What about for the pixel value? 
![image](https://github.com/user-attachments/assets/36fcc032-ad2a-4e51-a9d5-6c625ed625c8)

Reply:
To clarify, I was thinking of ways to shorten the pixel value, not the position. I think using `Math.round` for the position is good, since we don't need much more resolution beyond the decimal point.

Reply:
Got it now.